frontend/components/Editor/Editor.tsx

```diff
@@ -123,6 +167,7 @@ const Editor = ({
   return (
     <div className={classNames}>
       {renderLabel()}
+      {renderCopyButton()}
```

Comment:
Should this only render if `enableCopy` is `true`?

Reply:
whoops ty!